### PR TITLE
fix: add v0.0.2 subfolder for builder testdata

### PR DESCRIPTION
**Pull request by asraa**
Signed-off-by: Asra Ali <asraa@google.com>

https://github.com/slsa-framework/slsa-verifier/issues/73

### CODE REVIEW

**Review by laurentsimon — COMMENTED** [review]
I'm wondering:
1. For provenance_test.go: do we need per-version tests, or it's OK to only tests at HEAD?
2. For main_test.go: shall we re-generate tests so that they come from `slsa-framework/package-example`, and use a loop:
```
for v in versions:
  artifact = sprintf(./testsdata/%v/binary-blabla, v)
  source = github.com/slsa-framework/example-package
```

This way it will work for all versions. Wdut?

File: RELEASE.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ This is a  document to describe the release process for the verifier.
 - [Verify provenance](#verify-provenance)
 - [Update documentation](#update-documentation)
 - [Update builders](#update-builders)
+- [Add tests for new builder release version](#update-tests)
```
Comment (laurentsimon) [inline RELEASE.md:11]: should this not be in the builder's RELEASE.md? We already have it mentioned, but you can add the link to the workflows u created

**Review by laurentsimon — APPROVED** [review]
Thanks

File: RELEASE.md (hunk 1 above)
Comment (asraa) [reply RELEASE.md:11]: moving it!

File: main_test.go [hunk 2]
```diff
@@ -18,6 +19,9 @@ func pString(s string) *string {
 	return &s
 }
 
+// Versions of the builders to test
+var generatorVersions = []string{"v0.0.2"}
```
Comment (laurentsimon) [inline main_test.go:23]: how about v1.0.0?

Comment (laurentsimon) [reply main_test.go:23]: we're going to need it before we cut the release

Comment (asraa) [reply main_test.go:23]: If I added it right now, it'll fail because the data isn't there yet

**Review by laurentsimon — APPROVED** [review]

**Pull request merged**